genref/output/md/kube-proxy-config.v1alpha1.md

```diff
@@ -535,10 +535,10 @@ this always falls back to the userspace proxy.</p>
 
 - [KubeProxyConfiguration](#kubeproxy-config-k8s-io-v1alpha1-KubeProxyConfiguration)
 
-- [KubeSchedulerConfiguration](#kubescheduler-config-k8s-io-v1beta2-KubeSchedulerConfiguration)
-
 - [KubeSchedulerConfiguration](#kubescheduler-config-k8s-io-v1beta3-KubeSchedulerConfiguration)
 
+- [KubeSchedulerConfiguration](#kubescheduler-config-k8s-io-v1beta2-KubeSchedulerConfiguration)
```

Comment:
Not sure why this change was made (add newline)? Do you want to list `v1beta2` before `v1beta3`?

Reply:
No, I didn't manually sort this list.
Currently the list is randomly sorted.